transform-plugins/src/main/java/io/cdap/plugin/CloneRecord.java

```diff
@@ -59,6 +67,21 @@ public void prepareRun(StageSubmitterContext context) throws Exception {
     FailureCollector collector = context.getFailureCollector();
     config.validate(collector);
     collector.getOrThrowException();
+
+    Schema inputSchema = context.getInputSchema();
+    if (inputSchema == null || inputSchema.getFields() == null || inputSchema.getFields().isEmpty()) {
+      return;
+    }
+    Set<String> input = inputSchema.getFields().stream().map(Schema.Field::getName).collect(Collectors.toSet());
+
+    List<FieldOperation> operationList = new ArrayList<>();
+    for (String inputField : input) {
+      FieldTransformOperation operation =
+          new FieldTransformOperation("clone" + inputField, "Clone field " + inputField,
```

Comment:
Start sentence. This should be a easy to understand English statement

Reply:
Ack.